### PR TITLE
add small changes

### PR DESCRIPTION
**Pull request by bgruening**

### CODE REVIEW

File: source/pulsar.rst [hunk 1]
```diff
@@ -26,6 +27,8 @@ For example, on ``it02`` pulsar endpoint the fqdn hostname is: ``it02.pulsar.ga
 Pulsar configuration
 --------------------
 
+# is a fork needed?
```
Comment (bgruening) [inline source/pulsar.rst:31]: Do we need to fork here before?

File: source/pulsar.rst [hunk 2]
```diff
@@ -78,7 +81,9 @@ Pulsar configuration
        - { 'src': 'patches/params_submission.patch', 'basedir': '/opt/pulsar/venv/lib/python2.7/site-packages', 'state': 'present', 'backup': 'yes' }
        - { 'src': 'patches/hostname.patch', 'basedir': '/opt/pulsar/venv/lib/python2.7/site-packages', 'state': 'present', 'backup': 'yes' }
 
-   where the ``<rabbit_mq_usegalaxy.eu_url>`` is the RabbitMQ url, retrieved from useGalaxy.eu.
+## is the UUID patch still needed?
```
Comment (bgruening) [inline source/pulsar.rst]: ping?

**Review by bgruening — COMMENTED** [review]
@gmauro @mtangaro a few small edits. This is awesome! Thanks a lot for working on it.

I think a small intro text would be nice on the front-page.

File: source/pulsar.rst (hunk 2 above)
Comment (gmauro) [reply source/pulsar.rst]: Yes, John's UUID commit is available on GX20.01 and Pulsar0.14.dev1, which are both one step above what we are using now. Furthermore, it works with that GX version only. Instead, mine can be used also with previous versions.

**Review by gmauro — APPROVED** [review]

**Comment by bgruening** [comment]
Danke.

**Pull request merged**